packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/index.js

```diff
@@ -25,6 +25,18 @@ class ExperimentChoiceIndex {
     this._experimentGroupingRules =
       options.experimentGroupingRules || experimentGroupingRules;
     this._featureFlags = options.featureFlags;
+
+    // Apply configured rollout rates
+    if (options.rolloutRates) {
```

Comment:
Something seems funky here, it's been a while since I've set one of these up but we've had several experiments in the past with configurable rollout rates so I wouldn't think we'd need this... Did you look at [the docs](https://mozilla.github.io/ecosystem-platform/reference/experiments-ab-testing) we have or past experiments we've removed? 🤔 Maybe we do need this, just double checking.

Reply:
I looked at all the current experiments and docs and didn't see anything. They had hard coded rollout rates. I didn't look at previous experiments, but I did perused the experiments module and didn't find another way to set this. This actually a descent part of the effort, cause I sort of assumed there was a way to do this.